client/verta/docs/changelog.rst

```diff
@@ -37,6 +37,21 @@ Changelog
      (`# <>`__)
 
 
+v0.19.2 (2021-09-30)
+--------------------
+
+New Features
+^^^^^^^^^^^^
+- add ``RegisteredModelVersion.change_stage()``
+  (`#2654 <https://github.com/VertaAI/modeldb/pull/2654>`__)
+
+Bug Fixes
+^^^^^^^^^
+- put upper constraint on `protobuf version for Python 2 compatibility
```

Comment:
Whoops, missed a closing backtick after "protobuf".

Fixed for mainline in #2668